tests/unit/amo/components/TestFooter.js

```diff
@@ -115,5 +115,49 @@ describe(__filename, () => {
         utm_campaign: null,
       })}`,
     );
+
+    expect(root.find('.Footer-desktop-link')).toHaveText('Desktop');
+    expect(root.find('.Footer-desktop-link')).toHaveProp(
+      'href',
+      `https://www.mozilla.org/firefox/new/${makeQueryStringWithUTM({
+        utm_source: 'addons.mozilla.org',
+        utm_medium: 'referral',
+        utm_content: 'footer-link',
+        utm_campaign: null,
+      })}`,
+    );
+
+    expect(root.find('.Footer-mobile-link')).toHaveText('Mobile');
+    expect(root.find('.Footer-mobile-link')).toHaveProp(
+      'href',
+      `https://www.mozilla.org/firefox/mobile/${makeQueryStringWithUTM({
+        utm_source: 'addons.mozilla.org',
+        utm_medium: 'referral',
+        utm_content: 'footer-link',
+        utm_campaign: null,
+      })}`,
+    );
+
+    expect(root.find('.Footer-reality-link')).toHaveText('Reality');
+    expect(root.find('.Footer-reality-link')).toHaveProp(
+      'href',
+      `https://mixedreality.mozilla.org/${makeQueryStringWithUTM({
+        utm_source: 'addons.mozilla.org',
+        utm_medium: 'referral',
+        utm_content: 'footer-link',
+        utm_campaign: null,
+      })}`,
+    );
+
+    expect(root.find('.Footer-enterprise-link')).toHaveText('Enterprise');
+    expect(root.find('.Footer-enterprise-link')).toHaveProp(
+      'href',
+      `https://www.mozilla.org/firefox/enterprise/${makeQueryStringWithUTM({
+        utm_source: 'addons.mozilla.org',
+        utm_medium: 'referral',
+        utm_content: 'footer-link',
+        utm_campaign: null,
+      })}`,
```

Comment:
I think we can also simplify the test cases here (by omitting the default params)